testing/docker/rabbitmq_setup.go

```diff
@@ -21,7 +21,7 @@ var (
 )
 
 // SetupRabbitMQ sets up a RabbitMQ broker
-func SetupRabbitMQ(testName string) (string, error) {
+func SetupRabbitMQ(testName string, timeOut time.Duration) (string, error) {
```

Comment:
This is technically a breaking change. Can you rename this func to `SetupRabbitMQWithTimeout` and add a new func that calls this, e.g.:

```golang
func SetupRabbitMQ(testName string) (string, error) {
    return SetupRabbitMQWithTimeout(testName, 30*time.Second)
}
```

Reply:
I was on the fence about this since we _should_ be requiring timeouts for all of these, but alas this is an imperfect world and avoiding breaking changes is the more practical option

Reply:
Sure

Reply:
Or use the Daniel's proposed solution. It works equally well.